partials/instagram.php

```diff
@@ -1,12 +1,39 @@
-<?php 
+<section id="instagram-feed" class="row"> 
+
+<?php
 $instagram_handle = IGV_get_option( '_igv_instagram_handle' ); 
-?>
-<section class="row"> 
+if ($instagram_handle) {
```

Comment:
i left this here and send the handler as a parameter to the `get_instagram_feed()` function